### DISCUSSION TITLE
Minio Prod and DR warm site Practise with Site Replication

### DISCUSSION THREAD

**Discussion by Ijoezl** [minio/minio#20528] [Q&A] [answered]
Hi all,

I am going to start off by wishing you all the best with everything.

I have some doubts with Site Replication for minio and i need some guidance for this.

So i have Production Env Minio and Warm Site DR Minio (Both in Linux RHEL). The steps are as below.
1. During pre setup, i will configure Prod will be act as main and configure site replication from Prod to DR.
2. Also while pre setup DR will be up to test the site replication.
3. After the site replication is done, verified the site replication status connectivity and the bucket data has been sync. DR will be shutdown and will act as warm site DR Minio.

My question: Considering the Prod Minio will run actively with an amount of 5TB storage
a. Will the site replication link between Prod and DR will not be remove automatically after more than months or years of the DR will be shutdown when its not needed?

b. What about the constant queue replication request from Prod to DR while DR is turned off. Prod will actively being updated with new data and these new data will be stuck in queue replication from Prod to DR and this constant queue will keep on growing. Will this affect the performance of my Prod Minio badly? 

c. If DR site remains down for an extended period of time, it may encounter large backlogs of data needing replication when it comes back online later down the line. Will there by any problem to sync back from Prod? any suggestion to avoid this issue?

Appreciate your time and suggestion if any and in advance.

Thanks!

**Comment by klauspost** [comment, marked as answer]
Since you are looking to do a prod setup, I suggest you reach out on sales@min.io so we can give you direct and detailed guidance in your setup.

Overall observations: Replication isn't meant to be "shut down", and everything is designed about replication being continuous. As such I have no idea how well it would operate with the setup you are proposing. There is [batch replication](https://min.io/docs/minio/kubernetes/upstream/administration/batch-framework-job-replicate.html) which is more of a once-off thing. Reach out for details.
